tests/scripts/depends.py

```diff
@@ -261,7 +261,10 @@ def test(self, options):
                          'MBEDTLS_ENTROPY_FORCE_SHA256',
                          'MBEDTLS_SHA256_USE_A64_CRYPTO_IF_PRESENT',
                          'MBEDTLS_SHA256_USE_A64_CRYPTO_ONLY'],
-    'MBEDTLS_X509_RSASSA_PSS_SUPPORT': []
+    'MBEDTLS_X509_RSASSA_PSS_SUPPORT': [],
+    # The support for TLS 1.3 is not complete yet and in particular
+    # pre-shared keys are not supported.
+    'MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_EPHEMERAL_ENABLED': ['MBEDTLS_SSL_PROTO_TLS1_3']
```

Comment:
Minor, pre-existing: pleave leave a coma after the last entry, that makes future additions a bit easier.